bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/CCombo.java

```diff
@@ -2025,4 +2025,20 @@ public boolean traverse(int event){
 	}
 	return super.traverse(event);
 }
+
+/**
+ * The method accepts a combo and a callback which takes
+ * all the child of the CCombo as the argument and executes it.
+ * All children are refreshed after the execution of the callback.
+ *
+ * @noreference This method is not intended to be referenced by clients.
+ * @param combo
+ * @param childUpdater
```

Comment:
Please add documentation for the parameters

Reply:
@HeikoKlare Added!